docs/components/board/upboard.md

```diff
@@ -34,7 +34,46 @@ Enter a name for your board and click **Create**.
 
 ![An example configuration for a upboard board in the Viam app Config Builder.](/components/board/upboard-ui-config.png)
 
-Edit and fill in the attributes as applicable.
+Copy and paste the following attribute template into your board's **Attributes** box.
+Then remove and fill in the attributes as applicable to your board, according to the table below.
+
+{{< tabs >}}
+{{% tab name="Attributes template" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "digital_interrupts": [
+    {
+      "name": "<your-digital-interrupt-name>",
+      "pin": "<pin-number>",
+      "type": "< basic | servo >"
```

Comment:
```suggestion
      "type": "<basic|servo>"
```

Reply:
Only document this attribute (basic versus servo) for the raspberry pi.